### DISCUSSION TITLE
How to connect to a remote managed Alfen Box

### DISCUSSION THREAD

**Discussion by FerNerd** [leeyuentuen/alfen_wallbox#122] [Q&A]
Hi,

My Alfen box is part of a lease car contract and manged by 50five via a mobile connection. Is there a way to connect to the box to Home Assistant without disturbing that? 

Thanks!

**Comment by andi2690** [comment]
same here

**Comment by leeyuentuen** [comment]
my wall box is from Pluginvest (actually the same as Last Mile Solutions). They just get data over ocpp on mobile connection.
For me, I've added extra UTP from the wall charger to my network and I can open 1 HTTP connection to that wall box (that I'm using on home assistant). if you have that UTP cable I can also connect directly to my wallbox using the myeve app on my mobile phone and also with the ace installer application on windows

**Reply by phulstaert** [reply]
But only if you have the owner password, right?

**Reply by citoki** [reply]
> But only if you have the owner password, right?

The password for the charger itself, which can be used by various apps (MyEve, this HomeAssistant plugin, ACE) is printed on the manual.

You can get an ACE account here: https://knowledge.alfen.com/space/IN/135922077/How+do+I+receive+login+details+for+the+ACE+Service+Installer+configuration+tool%3F Needed to start ACE application (Windows)

**Reply by babavers** [reply]
> > But only if you have the owner password, right?
> 
> The password for the charger itself, which can be used by various apps (MyEve, this HomeAssistant plugin, ACE) is printed on the manual.
> 
> You can get an ACE account here: https://knowledge.alfen.com/space/IN/135922077/How+do+I+receive+login+details+for+the+ACE+Service+Installer+configuration+tool%3F Needed to start ACE application (Windows)

I have the same problem, also 50Five. I got ACE credentials and can login to the charger without any other specific owner's password. Also with the app. There is no charger password since it still runs the old 4.15.7-4054 firmware (see also [this page](https://knowledge.alfen.com/space/IN/243466257/Release+notes+-+Firmware+updates+NG-Platform+(NG-910+%252F+NG-920))). And I'm not going to upgrade that firmware risking to distroy the back office connection. 
Question is then how to connect with this integration? User and password is required in the requester. If I fill out anything, then all the sensor values are 'unknown'.

**Reply by citoki** [reply]
I also have backend integration configured and it was delivered with version 5, which was already 2 years old. I upgraded to the latest major 6 version and everything is still working fine. With the ACE you can do a backup before upgrading. Anyhow Alden strongly recommends to run the latest version.

**Reply by babavers** [reply]
> I also have backend integration configured and it was delivered with version 5, which was already 2 years old. I upgraded to the latest major 6 version and everything is still working fine. With the ACE you can do a backup before upgrading. Anyhow Alden strongly recommends to run the latest version.

Indeed, but it's problematic for me. If you upgrade from an older version to 5, then a password is generated during the update (probably based on the serial number or something) and the only way to get it is via the webstore of Alfen. That's what the release notes say. But I can't login to that webstore because an account is needed that can only be created by the owner of the machine which is the leasing company in my case.  So I'm stuck, I would make my charger completely inaccessible if I would do that. For the time being, I can only access it with the ACE installer that I used to change some parameters in a way that I can access it via modbus over ethernet. That way I can do something in HA (e.g. limit the max power), but it's very basic and difficult with a lot of trial and error. That's why I'm so interested in this package.

**Reply by citoki** [reply]
you could register at [Alfen aftersales portal](https://aftersales.alfen.com/plugins/servlet/desk/category/self-service) and ask for generating the password upfront with the needed information. Via "Manage your services" link in the portal you can click on "Request for Sign-Up" to get the password for the ACE Installer anyway. So it should be possible to get the password for your wallbox as well.

**Reply by babavers** [reply]
> you could register at [Alfen aftersales portal](https://aftersales.alfen.com/plugins/servlet/desk/category/self-service) and ask for generating the password upfront with the needed information. Via "Manage your services" link in the portal you can click on "Request for Sign-Up" to get the password for the ACE Installer anyway. So it should be possible to get the password for your wallbox as well.

I tried and guess what.... They sent me new credentials for the ACE Installer (which I already had of course). They don't even read my message with the slightest bit of attention...
